### ISSUE TITLE
Create simple network

### ISSUE THREAD

**Issue by itvai** [fchollet/keras#4055]
Hi,

i am new to keras and have a question it would be great if someone could help or at least give me some suggestions if my idea is stupid :) I am using keras with Tensorflow. 

I would like to build a supervised learning network. I have a .csv-file with data, in each row i have three input floats and one output float. The idea is to get the output float out of the three input floats. I have no classification for the output floats, they are in a range from about -50.00 to 50.00.

The network should react on the inputs so that that the results comme closer to the output. Thank you in forward for every idea.

**Comment by itvai** [comment]
I've build a network now, it takes some input and gives out some output. 

mode = Sequential()
model.add(Dense(1, input_dim=3))
modeladd(Actionvation = 'sigmoid'))
model.compile(loss='mse', optimizer='sgd')
model.fit(x,y, nb_epoch=100, batch_size=61)

My input is a [[a,b,c],[d,e,f]] list and my output is a [[g],[h]] list. 
Where a..f > 0.00 and -50 < g, h < 50.

I dont know how to get useful output of of it, does anyone know?
